common/src/androidTest/java/com/microsoft/identity/common/BrowserSelectorTest.java

```diff
@@ -253,15 +253,16 @@ public TestBrowserBuilder setVersion(String version) {
         }
 
         public TestBrowser build() {
-            PackageInfo pi = new PackageInfo();
-            pi.packageName = mPackageName;
-            pi.versionName = mVersion;
-            pi.signatures = new Signature[mSignatures.size()];
+            Signature [] signatures = new Signature[mSignatures.size()];
```

Comment:
nit: Do we typically put a space before the braces in an array type declaration?

Reply:
nope